roles/marathon/defaults/main.yml

```diff
@@ -12,7 +12,6 @@ marathon_java_settings: '-Xmx512m -Xms512m -XX:+HeapDumpOnOutOfMemoryError'
 marathon_artifact_store: 'file:///store'
 marathon_artifact_store_dir: '/etc/marathon/store'
 marathon_server_zk_group: marathon_servers
-marathon_rebuild_container: False
```

Comment:
should be recreateable by changing the image name (which will update the systemd file) and reload the service with the new image in place.